.yarn/versions/2f8bb76f.yml

```
@@ -0,0 +1,35 @@
+releases:
+  "@yarnpkg/cli": patch
+  "@yarnpkg/core": patch
+  "@yarnpkg/shell": patch
```

```suggestion
  "@yarnpkg/cli": feat
  "@yarnpkg/shell": feat
```

`@yarnpkg/core` doesn't need to be released for this.